src/py/flwr/client/supernode/app.py

```diff
@@ -63,6 +65,21 @@ def run_client_app() -> None:
 
     args = _parse_args_run_client_app().parse_args()
 
+    if args.server != ADDRESS_FLEET_API_GRPC_RERE:
+        warn = "Passing flag --server is deprecated. Use --superlink-fleet-api instead."
+        warn_deprecated_feature(warn)
+
+        if args.superlink_fleet_api != ADDRESS_FLEET_API_GRPC_RERE:
+            # if `--superlink-fleet-api` also passed, then
+            # warn user that this argument overrides what was passed with `--server`
+            log(
+                WARN,
+                "Both `--server` and `--superlink-fleet-api` were passed. "
+                "`--server` will be ignored.",
```

Comment:
Suggestion: "`--server` will be ignored. Connecting to the Superlink Fleet API at {args.superlink_fleet_api}."

Reply:
I added this.